### PR TITLE
getChunkLimitsExceededMessage : Allow specifying %Limit% and %Material%

### PR DESCRIPTION
**Pull request by squareblob**

### CODE REVIEW

File: src/main/java/com/programmerdan/minecraft/simpleadminhacks/hacks/GameTuning.java [hunk 1]
```diff
@@ -221,7 +221,10 @@ public void chunkLimits(BlockPlaceEvent event) {
 				if (state != null && mat.equals(state.getType())) {
 					if (++current > limit) {
 						event.setCancelled(true);
-						player.sendMessage(config.getChunkLimitsExceededMessage());
+						player.sendMessage(config.getChunkLimitsExceededMessage()
+								.replaceAll("%Limit%",Integer.toString(limit))
```
Comment (Maxopoly) [inline src/main/java/com/programmerdan/minecraft/simpleadminhacks/hacks/GameTuning.java:225]: Could you make these either all upper case (preferred) or all lower case? Capitalization like that usually isn't used in this context.

Comment (squareblob) [reply src/main/java/com/programmerdan/minecraft/simpleadminhacks/hacks/GameTuning.java:225]: I tried to follow the format set by other SAH config replacements

- [message: '%Victim% was combat tagged by %Attacker%'](https://github.com/CivClassic/AnsibleSetup/blob/fde59ac5a2c08470a3dae847544ee3168a986aff/templates/public/plugins/SimpleAdminHacks/config.yml.j2#L56)
- [announce: '&f%Player% is brand new!'](https://github.com/CivClassic/AnsibleSetup/blob/fde59ac5a2c08470a3dae847544ee3168a986aff/templates/public/plugins/SimpleAdminHacks/config.yml.j2#L60)

I can change it to all uppercase, should I edit all of SAH config replacements to be uppercase?

Comment (Maxopoly) [reply src/main/java/com/programmerdan/minecraft/simpleadminhacks/hacks/GameTuning.java:225]: No, just follow the convention then I guess. Weird.

**Comment by Maxopoly** [comment]
Thanka

**Pull request merged**